pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+[UV-metric pKa measurement protocol of Sirius T3](http://www.sirius-analytical.com/science/pka) evaluates pKas based on change of absorbance in the UV region of the spectrum while a pH titration is being performed between pH 1.8 and 12.2. UV-absorbance spectra of small molecule chromophores can change as a function of pH,  based on the protonation state change of groups close to the chromophore. Change in multiple wavelengths as a function of pH is fitted to a model to determine pKas. Number of pKas in the model is selected based on better fit between experimental and modeled microstates curves. 
+
+This method is capable of measuring pKas between 2 and 12, when protonatable groups are at a maximum distance of 4-5 atoms away from chromophores, so that a change of protonation state affects the absorbance of UV-chromophore. We have selected compounds where titratable groups are close to potential chromophores (conjugated rings), but it is possible to completely miss detection of a pKa value if a titratable group is not in interaction with a UV-chromophore.  
+
+pKa measurements of soluble compounds were performed in ionic-strength adjusted water with 0.15 M KCl. Indications for precipitation was observed by visual inspection of samples and also by inspection of UV-spectra plots as a function of pH at 500-600 nm region of the spectra. For compounds with insufficient solubility, cosolvent protocol is used where 3 UV-metric pKa measurements were done at different cosolvent:water ratios (typically 30%, 40% and 50% methanol) and  Yasuda-Shedlovsky extrapolation method is used to estimate pKa value at 0% cosolvent.
+
+Three replicate pKa measurements were made for all compounds at room temperature (25°C). Multiwavelength absorbance analysis of Sirius T3 allows very good resolution pKas but essentially this method measures macroscopic pKas. Microscopic pKas with with very close pKa values and overlapping changes in absorbance spectra could be measured as one macroscopic pKa value.
+
+## Instructions and Submission Templates
+Three types of predictions will be accepted. Participants are encouraged to submit their results in all or multiple submission types as it fits to their prediction methods.
+
+#### Type I - microscopic pKas and related microstates
+Predicting microscopic pKa values and related microstates structures. Different protonation states and tautomers constitute different microstates. 
+- Fill the `typeI_microscopic_pKas_and_microstates.csv` template for all molecules.
+- For each molecule, report as many microscopic pKas based on your best guest.
+- Record the pair of microstates IDs of microstate structures pairs (protonated HA and deprotonated A) associated for each microscopic pKa. To determine the microstate ID for your predicted structure check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
```

Comment:
```
Record the pair of microstate IDs of microstate structures pairs (protonated HA and deprotonated A) associated with each microscopic pKa. 
To determine the microstate ID for your predicted structure, check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates) that list all microscopic species.
```

Reply:
If they are csv files, I would not call them spreadsheets; they're text files which could be viewed as spreadsheets, right?

Reply:
I included csv files (microstate ID + SMILES) and excel spreadsheets (microstate ID + SMILES + 2D depiction). I changed the text to mention both.